Clarifications on SAS tokens in the intro

clemensv opened the pull request:
Clarified that SAS is a token model and that tokens can also be generated by an external service based on an external authorization check.

azurecla commented:
Hi __@gperrot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

clemensv committed:
Clarifications on SAS tokens in the intro 

Clarified that SAS is a token model and that tokens can also be generated by an external service based on an external authorization check.
